client/state/data-getters/index.js

```diff
@@ -209,3 +209,21 @@ export const requestSiteAlerts = siteId => {
 		}
 	);
 };
+
+export const requestAtomicHostingPmaNonce = siteId =>
+	requestHttpData(
+		`atomic-hosting-pma-nonce-${ siteId }`,
+		http(
+			{
+				method: 'GET',
```

Comment:
This should probably be a POST in final version as this call is not idempotent, but setting POST here currently forces this to call v1 endpoint instead of v2 for some reason.